airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -261,7 +261,7 @@ def get_stream_state_value(self, stream_state: Mapping[str, Any] = None) -> int:
         Returns the state value, if exists. Otherwise, returns user defined `Start Date`.
         """
         state = stream_state.get(self.cursor_field) or self._start_date if stream_state else self._start_date
-        return calendar.timegm(pendulum.parse(state).utctimetuple())
+        return int(state) if isinstance(state, int) or state.isdigit() else calendar.timegm(pendulum.parse(state).utctimetuple())
```

Comment:
I don't have an issue w/ the code, but i'm interested in how this got surfaced. Was there a customer that saw this as an issue or was this just found when you were writing mock server tests?

Reply:
It is not a bug. It is to have [this test](https://github.com/airbytehq/airbyte/pull/55676/files#diff-a3b061d62946cc122ae90fda95db6e1f75bd96492ee3adee79b836608798888cR138-R157) pass. However, there are so few users in prod that I don't think we intend to revert. We could just remove that honestly...